docs/docs/creating-and-modifying-pages.md

```diff
@@ -263,6 +263,23 @@ export default Page
 
 Page context is serialized before being passed to pages. This means it can't be used to pass functions into components and `Date` objects will be serialized into strings.
 
+### Optimizing pages for content sync
+
+If using the Preview UI Content Sync feature on Gatsby Cloud, an optional parameter, `ownerNodeId` can be passed to the `createPage` invocation to give you greater control over how content is previewed. By passing a value to `ownerNodeId`, you can ensure that the CMS content is properly mapped to the page you would like to preview it on. For example, the value of `ownerNodeId` can be set to `node.id` when pages are generated for each queried node.
```

Comment:
Can we link "Preview UI Content Sync" to somewhere?

Reply:
Agreed, it would be nice also to show the "Hierarchy of how we find a page owner", ie ownerNodeId, fileSystemRoute API, etc. I would assume that info would live somewhere else and we could just link to it.

Reply:
```suggestion
When using the Content Sync feature on Gatsby Cloud, an optional parameter, `ownerNodeId`, can be passed to the `createPage` action to allow greater control over where content is previewed. By passing a value to `ownerNodeId`, you can ensure that Content Sync will redirect content authors to the page they intend to preview their content on. The value of `ownerNodeId` should be set to the id of the node that's the preferred node to preview for each page. This is typically the same node that is used to create the page path for each page.
```

Reply:
Once we merge https://github.com/gatsbyjs/gatsby/pull/33866 we can reference that page here